# MATERIALS AND METHODS

## Soil origin and sampling.

The forest site is located within the Kohala Forest Reserve on the Big Island of Hawaii (20°03′N, 155°41′E) and has never been cleared by humans.
The forest site is the same as the Kohala site in the study of a chronosequence by @Crews1995 [@Crews1995].
The pasture site was cleared in the 1920's for ranchland and lies immediately adjacent to the forest.
The pasture was seeded with the African grass *Pennisetum clandestinum*, which now dominates this species-poor site.
Cattle graze the pasture.

The parent material of this Typic Placandept is a volcanic ash deposit, tephra, which was deposited 150,000 years ago.
The sampling sites are at 1,122-m elevation, which corresponds to a mean annual air temperature of 16°C [@UniHawaii1983].
The mean annual rainfall is near 2,500 mm [@Giambelluca1986].
The pristine closed canopy rain forest is dominated (83% of the cover) by a single species, the native tree *Metrosideros polymorpha*, the tree most widespread on the Hawaiian islands [@Kitayama1995].
The well-developed understory vegetation is dominated by native tree ferns of *Cibotiumspp.* (25% of the cover).
Other trees and shrubs at this site are those of the genus *Cheirodendron* (18% of the cover), understory trees of the genus *Coprosoma* (4% of the cover), and shrubs of the genus *Vaccinium* (2% of the cover), accounting for most of the remaining cover [@Crews1995].
The exotic plant species *Psidium cattleianum* and *Hedychium garnerianum* are also present in low abundance [@Riley1995].
The total number of vascular plant species per 0.2 ha is 53, while the number of native plant species is 45 [@Crews1995].
In temperate forests there are about 140 vascular plant species per 0.2 ha [@Huston1994].

Soil samples were collected in 1992, 1994, and 1999.
Forest samples were from sites located 30 to 50 m apart, and pasture samples were from sites that spanned 5 km.
Samples were taken from the upper 8- to 10-cm layer of the umbric A horizon after removal of the overlaying organic root and litter layer and placed immediately in sterile plastic bags on wet ice.
After 48 h samples were shipped on dry ice to our laboratory in Michigan and stored at −20°C.
The soil moisture content was determined by drying soil overnight at 100°C.
Soil mechanical and chemical analyses were done by the Soil Analysis Laboratory, Michigan State University, using the methods described in @Peck1988 [@Peck1988].

## DNA extraction, purification, and G+C fractionation.

DNA was extracted from 10 g of soil by using the direct lysis method of @Holben1994 with the following modifications: a lower shaker speed, an extended shaking time, a 100-fold-increased phosphate concentration (due to soil acidity), and no addition of EDTA to the lysis buffer [@Nusslein1998].
Extraction efficiency was determined by comparing the amount of DNA extracted with the expected amount of DNA calculated from the difference in direct microscopic counts of bacterial cells before and after lysis.
DNA was quantified by fluorometry [@Nusslein1998].
The amount of extracellular DNA was determined following extraction with sodium phosphate before lysis (data not shown).

DNA fragments were separated according to G+C content following the procedure of @Holben1995 [@Holben1995].
Briefly, DNA was mixed with the A- and T-specific dye bis-benzimidazole, and the buoyant density of the resulting DNA–bisbenzimide complex was decreased in proportion to the amount of dye bound.
A gradient of percent G+C concentration was then established by equilibrium density gradient ultracentrifugation.
The amount of DNA in each fraction was determined by absorption spectroscopy.
To allow PCR amplification DNA fractions were purified from the CsCl and the dye by repeated extractions in CsCl-saturated isopropanol followed by spin column chromatography (Wizard PCR MiniPrep; Promega, Madison, Wis.) as previously described [@Nusslein1998].

## rRNA analysis.

G+C fractions (with 63 and 35% G+C contents) derived from two subsamples of soil from composite samples representing a 20-cm<sup>2</sup> area each of forest and pasture were used to produce replicate SSU rRNA clone libraries.
SSU rRNA genes were amplified by PCR from the DNA fractions by using the eubacterium-specific primers fD1 and rP2 of @Weisburg1991 [@Weisburg1991].
PCRs were performed with *Taq* DNA polymerase (Boehringer Mannheim, Indianapolis, Ind.) according to the manufacturer's protocol.
To ensure that only soil bacterial rRNA genes were amplified the following quality control steps were implemented: determination of primer purity by using high-performance liquid chromatography, preparation of fresh oligonucleotide primers for each use, a preamplification heating step to maximize PCR sensitivity and specificity, and the use of stock solutions for negative controls [@Nusslein1998].
Amplified products were separated in agarose gels, and the bands were visualized by UV excitation after ethidium bromide staining.
Prior to cloning, the amplified SSU rDNA fragments were purified by spin column chromatography (Wizard PCR Miniprep; Promega) and an equimolar amount of amplified PCR products was ligated to the vector pCR II (Invitrogen Corp., Carlsbad, Calif.) and transformed into *Escherichia coli* Top-10F′ competent cells.
A primer pair specifically designed to complement the polylinker of the vector pCR II was used to amplify plasmid inserts directly from the transformant cells for SSU rDNA gene screening [@Zhou1997].
To screen for SSU rDNA diversity, the amplified inserts were digested with two sets of tetrameric restriction endonucleases, *Hae*III with *Hha*I and *Msp*I with *Rsa*I [@Nusslein1998].
The resulting fragments were electrophoretically resolved, and the similarities between the electrophoretic patterns were analyzed by using GelCompar (Applied Mathematics, Kortrijk, Belgium) cluster analyses by comparative numerical analysis with unweighted pair-group method using arithmetic averages (UPGMA).
Individual clones were grouped by using a cutoff of 97% similarity and a 5% error rate for the band position.
Each different SSU rRNA pattern is termed a phylotype.

## Determination of nucleotide sequences and phylogenetic analysis.

Purified SSU rDNA clone inserts were sequenced in both directions by the fluorescent DiDeoxy termination method by using automated fluorescent *Taq* cycle sequencing on the ABI Catalyst 800 and ABI 373A sequencing system (Applied Biosystems, Foster City, Calif.).
Partial sequences were done for the first 500 bp of the 5′ region (*E. coli*; numbering positions 11 through 529), and the dominant and novel types were fully sequenced.
All sequences were aligned, and phylogenetic relationships were inferred as described previously [@Nusslein1998].
Only unambiguously aligned nucleotide positions were used for the sequence analysis.
Potential chimeric artifacts were evaluated by CHECK_CHIMERA [@Maidak1999] and mglobalCHI [@Komatsoulis1997] and by comparing the phylogenetic affiliations of the 5′ end of a 16S rRNA gene with those of the 3′ end.
All phylogenetic nomenclature was based on the RDP-II version 7.0 [@Maidak1999].

## Nucleotide sequence accession numbers.

The sequence data for the Hawaiian rain forest soil clones HRS-46, HRS-47, HRS-50, HRS-55, HRS-56, and HRS-60 and the Hawaiian pasture soil clones HPS-45, HPS-49, HPS-52, HPS-54, HPS-61, and HPS-64 have been deposited in the GenBank database under accession nos. AF165267 to AF165280.
